### ISSUE TITLE
Strategies for reducing false positives at the beginning of a fine-tuning cycle.

### ISSUE THREAD

**Issue by bw4sz** [weecology/BOEM#9]
As we explore other architectures and ideas, here is a problem we need to solve.

The first rounds of fine tuning for any new flights consist basically of false positive detections the model encounters a new background. There are very few objects of interest. Training a model on empty images is quite unstable for the retinanets, the loss drives to 0 with any real learning rate. Is this architecture specific? Should we just keep retraining the entire backbone with new false positive data? Part of the issue is that DeepForest validation is working when there is 100% empty data.
